src/services/mentees.js

```diff
@@ -902,7 +915,7 @@ module.exports = class MenteesHelper {
 			let organizationIds = []
 			filterType = filterType.toLowerCase()
 			const attributes =
-				filterType == common.MENTEE_ROLE
+				filterType == common.MENTEE_ROLE || common.SESSION
```

Comment:
If its session, we should get the session policies right?

Reply:
@nevil-mathew , external_mentee_visbility === external_session_visibility 
so based on that i did migrations also , so checking mentee is also fine it is same as session